OpenRomu/GameConfig.cfg

```
@@ -8,7 +8,8 @@
         "searchpath": "valve",
         "packageformat": { "extension": "pak", "format": "idpak" }
     },
-    "textures": {
+    "materials": {
+		"root": "textures",
```

```suggestion
        "root": "textures",
```
This formatting would be more accurate